.github/workflows/ci.yml

```diff
@@ -302,6 +302,14 @@ jobs:
           POSTHOG_API_KEY: ${{ secrets.POSTHOG_API_KEY }}
         run: cd docs/docusaurus && yarn install && bash ../build_docs
 
+      - name: Run broken link checker
```

Comment:
Are we still running the custom script to check the broken links?

Reply:
Yes, we are still running it, i just added this as another validation, we could delete the other script in a different PR